src/content/en/updates/2016/12/lighthouse-dbw.md

```diff
@@ -0,0 +1,87 @@
+{: .note }
+
+<img src="/web/progressive-web-apps/images/pwa-lighthouse.png" class="lighthouse-logo attempt-right" alt="Lighthouse Logo">
+
+Since Google I/O, we've been hard at work making Lighthouse an awesome companion for building great [Progressive Web Apps](/web/progressive-web-apps/):
+
+- Welcomed 50 new [contributors](https://github.com/GoogleChrome/lighthouse/graphs/contributors) to the project
+- Shipped 15 [releases](https://github.com/GoogleChrome/lighthouse/releases)
+- Added ~20 additional [audit tests](https://github.com/GoogleChrome/lighthouse/tree/master/lighthouse-core/audits) (~50 total)
+
+Today, we're happy to announce the 1.3 release of Lighthouse. Lighthouse 1.3 includes a bunch of big new features, audits, and the usual bug fixes. You can install it from npm (npm i -g lighthouse) or [download the extension](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk) from the Chrome Web Store.
+
+So what's new?
+
+## New look and feel
+
+If you've used an earlier version of Lighthouse, you may have noticed that the logo is new! The HTML report and Chrome Extension have also undergone a complete refresh, with a cleaner presentation of scoring and more consistency across audit results. We've also added helpful debug information when you fail a test and include pointers to recommended workarounds.
+
+<img src="/web/updates/images/2016/12/lighthouse-dbw/report.png" class="screenshot" alt="Lighthouse Report">
+
+## New Best Practices
+
+To date, Lighthouse has focused on performance metrics and the quality of PWAs. However, an overarching goal of the project is to provide a guidebook for all of web development. This includes guidance on general best practices, performance and accessibility tips, and end-to-end help on making quality apps. 
+
+"Do Better Web" is an effort within the Lighthouse project to help developers do better on the web. In other words, help them modernize and optimize their web applications. Oftentimes, web devs use outdated practices, antipatterns, or hit known performance pitfalls without realizing it. For example, it is [widely known](https://www.html5rocks.com/en/tutorials/speed/animations/) that JS-based animations should be done with [`requestAnimationFrame()`](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) instead of [`setInterval()`](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setInterval). However, if the developer is unaware of the newer API, their web app needlessly suffers.
```

Comment:
Instead of pointing to https://www.html5rocks.com/en/tutorials/speed/animations/ - can we point to internal content - for example: https://developers.google.com/web/fundamentals/design-and-ui/animations/